docs/reference/tab-widgets/inference-api/infer-api-search.asciidoc

```diff
@@ -72,6 +72,68 @@ query from the `cohere-embeddings` index sorted by their proximity to the query:
 
 // end::cohere[]
 
+// tag::elser[]
+
+[source,console]
+--------------------------------------------------
+GET elser-embeddings/_search
+{
+  "query":{
+    "text_expansion":{
```

Comment:
Please replace this example with a `sparse_vector` query.

Reply:
Updated and tested. Changes have been pushed via [d40646](https://github.com/elastic/elasticsearch/pull/111406/commits/5d406464b6ff5924ccc506fb1118fd1f5c72b93c).